app/assets/stylesheets/application.scss

```diff
@@ -19,24 +19,40 @@
 @import "bootstrap";
 
 #body-wrapper {
-    width: 1050px;
+    max-width: 1050px;
     margin: 0 auto 3em;
 }
 
-nav ul li {
-    text-align: center;
+.nav-tabs {
+    align-items: stretch;
+    display: flex;
+    justify-content: space-between;
+
+    & > li {
+        display: flex;
+        padding: 0 5px;
+        text-align: center;
+        min-width: 20%;
+        & > a {
+            align-items: center;
+            display: flex;
+            flex-flow: row wrap;
+            justify-content: space-around;
+            padding: 10px 5px;
+            width: 100%;
+        }
+    }
 }
 
 footer {
+    align-items: center;
     background-color: #eee;
+    display: flex;
+    justify-content: space-around;
     width: 100%;
     position: fixed;
     bottom: 0;
-    height: 3em;
```

Comment:
Dropping the height value works, but it'd be cool to conditionally keep it with a media query like:
```
@media only screen and (max-device-width: 500px) {
    footer {
        height: auto;
    }
}

@media only screen and (min-device-width: 501px) {
    footer {
        height: 3em;
    }
}
```